tests/package-custom-coverage-test.js

```diff
@@ -140,7 +140,7 @@ describeApplication('PackageCustomCoverage', () => {
 
       describe('entering valid coverage', () => {
         beforeEach(() => {
-          return convergeOn(() => {
+          return new Convergence(() => {
```

Comment:
The API for `Convergence` is `new Convergence(timeout)`. However `@bigtest/mocha` will automatically add this timeout. Instead, these should be:

```
return new Convergence().once(() => {
  // ... expectations
});